pingserver/compare.py

```diff
@@ -0,0 +1,58 @@
+from subprocess import check_output
+import re
+import matplotlib.pyplot as plt
+import csv
+import pandas
+import os
+import time
+filename = "responsetime.csv"
```

Comment:
yeah, I get it's a filename, but of what? This should be more specific (particularly given you have a function that has a parameter called `filename` - don't shadow variables). call it `foo_filename` for whatever `foo` is.

Reply:
renamed to responsetime_filename